docs/source/getting_started/welcome_tour.rst

```diff
@@ -122,7 +123,7 @@ Putting all the pieces together, our trainer looks something like this:
 
 .. code-block:: python
 
-    state = State(...)
+    state = State(model, max_duration, rank_zero_seed, dataloader)
```

Comment:
Can probably keep this ambiguous (`...`) since these variables were not created previously.

Reply:
Okay, I could also create them above? I added them because I saw the dataloader was loading `state.targets` and `state.inputs`, so that led me to wonder how I get the `State` to attach to the dataloader.

Reply:
Ah yes agreed create them above would make sense.